packages/apps/web-viewer-test/src/PropertyGrid.scss

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
is this just for testing? We shouldn't be adding any property grid code to this repo aside from removing the widget from DefaultFrontstage and initializing the UIProvider instead